python/src/lib.rs

```diff
@@ -111,10 +110,13 @@ fn glide(_py: Python, m: &PyModule) -> PyResult<()> {
             Value::SimpleString(str) => Ok(str.into_py(py)),
             Value::Okay => Ok("OK".into_py(py)),
             Value::Int(num) => Ok(num.into_py(py)),
-            Value::BulkString(data) => match std::str::from_utf8(data.as_ref()) {
-                Ok(val) => Ok(val.into_py(py)),
-                Err(_err) => Err(PyUnicodeDecodeError::new_err(data)),
-            },
+            Value::BulkString(data) => {
+                // TODO: for now, and in order to keep the current tests to work,
```

Comment:
Just to confirm - you'll resolve this TODO in the next PR, right?
If yes, please add this to node and java clients too.

Reply:
Yes, its a multiple stage PR. but is getting too big, so we decided to break it into smaller PRs